### PR TITLE
Feature 68 event page styling

### LINKED ISSUE #68: Event page styling
as a user I want to see the events on mobile and desktop devices

### PR DESCRIPTION
**Pull request by Kaemppi**
resolves #68

### CODE REVIEW

File: src/components/pages/SingleEventPage.tsx [hunk 1]
```diff
@@ -105,9 +108,7 @@ class SingleEventPage extends React.Component<Props, State> {
       return null;
     } else {
       return (
-        <div>
-          <img className={classes.image_styles} src={fetchedContent.images[0].url} alt=""></img>
-        </div>
+        <img className={classes.image_styles} src={fetchedContent.images[0].url} alt=""></img>
```
Comment (Eskilmar) [inline src/components/pages/SingleEventPage.tsx:111]: There should be default picture, if event has no image ( https://mantyharju.fi/tapahtumat )

File: src/components/pages/WelcomePage.tsx [hunk 2]
```diff
@@ -356,7 +356,10 @@ class WelcomePage extends React.Component<Props, State> {
       return (
         parsedContent.splice(0, this.state.linkedEventsLimitingNumber).map(contentItem => {
           return (
-            <figure className={ classes.events_item_universal }>
+            <figure
+              // FIXME: linkedevents link should be here on the wrapping element
```
Comment (Eskilmar) [inline src/components/pages/WelcomePage.tsx:360]:
```suggestion
```

**Review by Eskilmar — CHANGES_REQUESTED** [review]

File: src/components/pages/SingleEventPage.tsx (hunk 1 above)
Comment (Kaemppi) [reply src/components/pages/SingleEventPage.tsx]: now the element is completely hidden if there is no image - I think no default image is needed

File: src/styles/single-event-page.ts [hunk 3]
```diff
@@ -152,20 +167,13 @@ export default createStyles({
...
-    marginTop: "1rem",
-    [breakpoints.up("sm")]: {
-      marginTop: "2rem"
-    },
-    [breakpoints.up("md")]: {
-      marginLeft: "2rem",
-      marginTop: 0
-    }
+
   },
```
Comment (Daniilmass) [inline src/styles/single-event-page.ts]: Consider removing empty rules

File: src/styles/single-event-page.ts [hunk 4]
```diff
@@ -152,20 +167,13 @@ export default createStyles({
     }
   },
   eventRightColumn: {
```
Comment (Daniilmass) [inline src/styles/single-event-page.ts:169]: Consider removing empty style blocks

File: src/styles/single-event-page.ts [hunk 5]
```diff
@@ -152,20 +167,13 @@ export default createStyles({
...
-    flex: 1
+
   },
   eventImageWrapper: {
 
   },
```
Comment (Daniilmass) [inline src/styles/single-event-page.ts]: Consider removing empty style blocks

File: src/styles/single-event-page.ts [hunk 6]
```diff
@@ -152,20 +167,13 @@ export default createStyles({
...
   eventDescription: {
```
Comment (Daniilmass) [inline src/styles/single-event-page.ts:175]: Consider removing empty style blocks

**Review by Daniilmass — CHANGES_REQUESTED** [review]

File: src/components/generic/Header.tsx [hunk 7]
```diff
@@ -120,7 +136,7 @@ class Header extends React.Component<Props, State> {
             </IconButton>
           </div>
           <Hidden smDown implementation="css">
-            <div className={ classes.menuWrapper } onMouseLeave={() => { this.onMouseLeave(); }}>
+            <div className={ menuWrapperStyles} onMouseLeave={() => { this.onMouseLeave(); }}>
```
Comment (Eskilmar) [inline src/components/generic/Header.tsx:139]:
```suggestion
            <div className={ menuWrapperStyles } onMouseLeave={() => { this.onMouseLeave(); }}>
```

File: src/components/pages/WelcomePage.tsx [hunk 8]
```diff
@@ -664,7 +671,7 @@ class WelcomePage extends React.Component<Props, State> {
     } else {
       const parsedContent = ReactHtmlParser(newsPost.content ? newsPost.content.rendered || "" : "");
       return (
-        parsedContent.splice(0, 4).map(contentItem => {
+        parsedContent.splice(0, 4).map((contentItem) => {
           return (
             <div className={ classes.allPosts}>
```
Comment (Eskilmar) [inline src/components/pages/WelcomePage.tsx:676]:
```suggestion
            <div className={ classes.allPosts }>
```

**Pull request merged**